bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -1011,6 +1011,11 @@ public void scanEntryLog(long entryLogId, EntryLogScanner scanner) throws IOExce
                 // read the entry
 
                 data.clear();
+                if (entrySize < 0) {
```

Comment:
as we don't know root cause, does it make sense to add a flag to skip entry-log on such failure? It can be helpful to users who don't want to skip and validate actual corruption.?